src/akamai/js/akamaiHD_flash.js

```diff
@@ -0,0 +1,1334 @@
+      break;
+    }
+  }
+  if (!pluginPath) {
+    console.error("Can't get path to script", filename);
+    return;
+  }
+  pluginPath += "akamaiHD_flash.swf";
+  var flexPath = "playerProductInstall.swf";
+  this.ready = false;
+  /**
+   * @class OoyalaFlashVideoFactory
+   * @classdesc Factory for creating video player objects that use Flash in an HTML5 wrapper.
+   * @property {string} name The name of the plugin
+   * @property {boolean} ready The readiness of the plugin for use.  True if elements can be created.
+   * @property {object} encodings An array of supported encoding types (ex. m3u8, mp4)
+   */
+
+  var OoyalaFlashVideoFactory = function() {
+    this.name = pluginName;
+    // This module defaults to ready because no setup or external loading is required
+    this.ready = true;
+    /**
+     * Checks whether flash player is available
+     * @public
+     * @method getFlashVersion
+     * @returns encoding as hds if flash version is available
+     */
+    function getFlashVersion() {
+      // ie
+      try
+      {
+        try
+        {
+          var axo = new ActiveXObject('ShockwaveFlash.ShockwaveFlash.6');
+          try
+          {
+            axo.AllowScriptAccess = 'always';
+          }
+          catch(e)
+          {
+           return '6,0,0';
+          }
+        }
+        catch(e) {}
+        return new ActiveXObject('ShockwaveFlash.ShockwaveFlash').GetVariable('$version').replace(/\D+/g, ',').match(/^,?(.+),?$/)[1];
+        // other browsers
+      }
+      catch(e) {
+        try
+        {
+          if (navigator.mimeTypes["application/x-shockwave-flash"].enabledPlugin) {
+            return (navigator.plugins["Shockwave Flash 2.0"] || navigator.plugins["Shockwave Flash"]).description.replace(/\D+/g, ",").match(/^,?(.+),?$/)[1];
+          }
+        }
+        catch(e) {}
+      }
+      return '0,0,0';
+    }
+
+    function testForFlash() {
+      var version = getFlashVersion().split(',').shift();
+      if (version < 11) {
+        console.error("NO FLASH DETECTED");
+        return [];
+      }
+      else {
+        return [ OO.VIDEO.ENCODING.AKAMAI_HD2_VOD_HDS ];
+      }
+    }
+    this.encodings = testForFlash();
+    this.technology = OO.VIDEO.TECHNOLOGY.FLASH;
+    this.features = [ ];
+
+    /**
+     * Creates a video player instance using OoyalaFlashVideoWrapper.
+     * @public
+     * @method OoyalaFlashVideoFactory#create
+     * @param {object} parentContainer The jquery div that should act as the parent for the video element
+     * @param {string} id The id of the video player instance to create
+     * @param {object} controller A reference to the video controller in the Ooyala player
+     * @param {object} css The css to apply to the video element
+     * @returns {object} A reference to the wrapper for the newly created element
+     */
+    this.create = function(parentContainer, id, controller, css) {
+      var video = $("<video>");
```

Comment:
In that case I think it might be lighter to make a simple <div> or something - in particular I think it may be confusing to find video elements where we don't expect them both for customers and us while debugging.  I propose that we change this but am interested in your thoughts